packages/next/server-actions/index.ts

```diff
@@ -35,14 +35,15 @@ export const signIn = async (
  */
 export const handleSignIn = async (
   config: LogtoNextConfig,
-  searchParams: URLSearchParams
+  searchParams: URLSearchParams,
+  redirectUri?: string
```

Comment:
should be one single param? e.g. `searchParams: URLSearchParam | URL` (you can use function override for better documenting)